dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -60,49 +64,74 @@
 <script lang="ts">
 import { defineComponent } from "vue";
 import PrimeButton from "primevue/button";
-import { CompanyReport } from "@clients/backend";
 import ReportFormElement from "@/components/forms/parts/ReportFormElement.vue";
 import ElementsDialog from "@/components/general/ElementsDialog.vue";
 import { ReportToUpload, StoredReport } from "@/utils/FileUploadUtils";
 import UploadDocumentsForm from "@/components/forms/parts/elements/basic/UploadDocumentsForm.vue";
+import { YesNoFormFieldProps } from "@/components/forms/parts/fields/FormFieldProps";
+import { CompanyReport } from "@clients/backend";
+import { ObjectType } from "@/utils/UpdateObjectUtils";
 
 export default defineComponent({
   name: "UploadReports",
+  inheritAttrs: false,
```

Comment:
Why is this necessary here?

Reply:
No sure. @MarcinStanczyk7 do you know the reason?

Reply:
👍